### PR TITLE
MUSARK-853

### PR DESCRIPTION
**Pull request by lasjo**
Oppdatert meldingene som gis ved scanning

### CODE REVIEW

File: language.json [hunk 1]
```diff
@@ -45,11 +45,11 @@
         "scanner": {
           "title": "Search for node or object",
           "search": "Search",
-          "noMatchingNodeOrObject": "No matching node or object for %{barcode}",
-          "noMatchingNode": "No matching node for %{uuid}",
-          "cannotActOnObject": "Can only act on barcode scans for objects from magasin or object picklist",
-          "cannotActOnNode": "Can only act on UUID scans from magasin, node picklist or move dialog",
-          "noCurrentLocation": "Object has no location"
+          "noMatchingNodeOrObject": "No matching node or object with barcode %{barcode}.",
+          "noMatchingNode": "No matching node with UUID %{uuid}.",
```
Comment (ingarabr) [inline language.json]: Do we want to expose the user for the type UUID? Will it be better to change it to identifier or something similar?

File: language.json [hunk 2]
```diff
@@ -694,11 +694,11 @@
         "scanner": {
           "title": "Søk etter node eller objekt",
           "search": "Søk",
-          "noMatchingNodeOrObject": "Fant ingen node eller objekt for %{barcode}",
-          "noMatchingNode": "Fant ingen node for %{uuid}",
-          "cannotActOnObject": "Kan bare reagere på object via magasin eller objekt plukkliste",
-          "cannotActOnNode": "Kan bare reagere på node fra magasin, node plukkliste eller flyttedialog",
-          "noCurrentLocation": "Objektet er ikke plassert"
+          "noMatchingNodeOrObject": "Fant ingen node eller objekt med strekkode %{barcode}.",
+          "noMatchingNode": "Fant ingen node med UUID %{uuid}.",
```
Comment (ingarabr) [inline language.json]: Same her for UUID.

Comment (jarlah) [reply language.json]: Good points. But we dont have any other identifier in this case. We dont know anything other than the provided UUID or barcode.

File: language.json (hunk 1 above)
Comment (lasjo) [reply language.json]: We could use  barcode instead

Comment (lasjo) [reply language.json]: OR drop both type and UUID

File: language.json (hunk 2 above)
Comment (lasjo) [reply language.json]: Same here

**Review by jarlah — APPROVED** [review]
LGTM

**Pull request merged**